go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/charmbracelet/glamour v0.7.0
 	github.com/charmbracelet/lipgloss v0.10.1-0.20240413172830-d0be07ea6b9c
 	github.com/cli/go-gh/v2 v2.11.0
+	github.com/cli/go-internal v0.0.0-20241025142207-6c48bcd5ce24
```

Comment:
This is one of those weird circumstances I don't know if we should preserve the release versions of the base repo when forking or not. 🤷 

If we plan to long term maintain the fork, then I suppose resetting the version number is okay.

Reply:
To be honest I think we planned to long term maintain our own version of testscript I would copy paste the relevant bits into `cli/cli`. 

Since we don't have a specific plan right now, I'm taking the path of least resistance (and I would have got away with it too if it weren't for those blasted Go maintainers!)

Reply:
![unmask-scoobydoo](https://github.com/user-attachments/assets/10ef4fb0-e4db-4975-abf3-d6bbf896850f)

Reply:
> This is one of those weird circumstances I don't know if we should preserve the release versions of the base repo when forking or not. 🤷 
> 
> If we plan to long term maintain the fork, then I suppose resetting the version number is okay.  On the other hand, it doesn't help us understand whether we're in sync with upstream changes on top of our customizations, which is why I'd suggest keeping the upstream versioning.